config/routes.rb

```diff
@@ -10,4 +10,9 @@
     resources :posts, only: %i[show index]
   end
 
+  resources :posts do
+      resources :comments, only: %i[create new]
+      resources :likes, only: %i[create] 
+  end
```

Comment:
- [ ] I couldn't find where to create a `new post` in your application and you do not have the `create` and `new` routes for the `create` and `new` actions in the posts controller which are responsible for creating a new post in your application. Kindly fix this so that a user can create a post successfully as it is part of the project requirement. I was able to create a post to test the comment and like through the seed you have in your seed file.